src/TTreeWidget.h

```diff
@@ -60,12 +63,22 @@ class TTreeWidget : public QTreeWidget
     void setIsKeyTree();
     void beginInsertRows(const QModelIndex& parent, int first, int last);
     void getAllChildren(QTreeWidgetItem*, QList<QTreeWidgetItem*>&);
+    QUndoStack* mpUndoStack = nullptr;
+    QTreeWidgetItem* mPrevParentItem;
+    QTreeWidgetItem* mParentItem;
+    QTreeWidgetItem* mdroppedItem;
+    dlgTriggersMainArea* mpTriggersMainArea = nullptr;
+    dlgTriggerPatternEdit* mpPatternItem;
+    dlgTriggerPatternEdit* mpTriggerPattern;
```

Comment:
Initialise every pointer that needs it - even if it is to a `nullptr` as that helps to make things go :boom: more reliably if it gets used without being pointed at the correct something!
```suggestion
    QTreeWidgetItem* mPrevParentItem = nullptr;
    QTreeWidgetItem* mParentItem = nullptr;
    QTreeWidgetItem* mdroppedItem = nullptr;
    dlgTriggersMainArea* mpTriggersMainArea = nullptr;
    dlgTriggerPatternEdit* mpPatternItem = nullptr;
    dlgTriggerPatternEdit* mpTriggerPattern = nullptr;
```